Traceback (most recent call last):
  File "C:\Users\acbart\Projects\datasets\builder\build.py", line 47, in build_dir
    output.write(data)
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2019' in position 8397: ordinal not in range(128)
